### PR TITLE
Docker Dektop Example

### PR DESCRIPTION
**Pull request by 1ly1mk**
This PR add docker desktop example for windows (can be also deployed in linux docker) with minimal user configuration. I believe that community will find this PR helpful.

### CODE REVIEW

**Comment by danduk82** [comment]
Hi @1ly1mk ,
could you please apply the suggested patch from pre-commit and create a new PR? Please use another branch name instead of `master` , this creates some side effects with the CI.

**Review by danduk82 — CHANGES_REQUESTED** [review]
Hi @1ly1mk ,
could you please apply the suggested patch from pre-commit and create a new PR? Please use another branch name instead of `master` , this creates some side effects with the CI.

**Review by danduk82 — APPROVED** [review]
Thanks @1ly1mk  for the contribution :)

**Comment by 1ly1mk** [comment]
Thank you @danduk82 , will try to create few more PR with windows based examples.

File: examples/docker-dektop/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+```
+helm upgrade --install ingress-nginx ingress-nginx \
+  --repo https://kubernetes.github.io/ingress-nginx \
+  --namespace ingress-nginx --create-namespace
+```
+
+## Navigate to docker-desktop/datadir and run helm install
+
+```
+ helm upgrade --install --values=.\values.yaml ogsdocker .
```
Comment (1ly1mk) [inline examples/docker-dektop/README.md:23]: @danduk82 just noticed, there typo with the release name it should be "ogsv1"
helm upgrade --install --values=.\values.yaml ogsv1 .

**Pull request merged**